pkg/pillar/cmd/zedagent/localinfo.go

```diff
@@ -137,17 +135,126 @@ func sendLocalAppInfo(ctx *getconfigContext) {
 				//throttle sending to be about once per hour
 				updateLocalAppInfoTicker(ctx, true)
 			}
-			if resp.StatusCode != http.StatusOK {
+			if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusNoContent {
```

Comment:
Would it make sense to instead have a switch on StatusCode?

Reply:
done